modules/photometric_calib/src/Reader.cpp

```diff
@@ -0,0 +1,81 @@
+//
```

Comment:
Hi Vincent,

I think there is a shorter version (basically a link to http://opencv.org/license.html) and the complete OpenCV's license, which one should I use?